09-schema/00-overview.md

```diff
@@ -5,29 +5,85 @@ permalink: /docs/schema/overview
+
+With Grakn, thanks to our high level knowledge model, your schema will closely resemble the conceptual schema, essentially avoiding the hassle of going through the other two modelling steps. The Grakn system will take care of those.
+
+This greatly simplifies the design process, getting you what can be considered a highly normalised distributed schema without the need of going through the logical and physical modelling.
+
+Let us have a look at the main components of a Grakn knowledge graph and schema.
+
+## Grakn Data Model
+
+### Concepts
+Everything that describes our domain in a Grakn Knowledge Graph is a concept. This includes the elements of the schema (namely types and roles, which we call schema concepts) and the actual data instances. 
+We refer to data instances as things - they can be thought of as instances of types defined in the schema.
+
+### Types
+Types constitute the core of your schema. They provide the necessary vocabulary to talk about our domain. They come in three flavours: [Entities](/docs/schema/concepts#entity), [Relationships](/docs/schema/concepts#relationship), and [Attributes](/docs/schema/concepts#attribute):
+
+__Entities__ are the main actors in our domain. These are usually the type of things we want to know about. Entity types provide means of classifying the objects in our domain.
 
-Lastly, we have the [**Graql Rules**](/docs/schema/rules). Rules watch for schema-driven patterns in the data and infer queryable conclusions. Rules are one way to perform automated reasoning in a Grakn knowledge graph.
+__Relationships__ are things that connect other concepts. Each relationship can connect a number of things, as specified in your schema. The character of participation in a relationship is characterised by roles that can be played in that relationship. Each relationship is required to
+have at least one role.
+
+__Attributes__ are used to characterise concepts with small pieces of data to other (think of numbers, strings, dates etc.). They allow to attach values of specified datatype to our instances.
+
+Apart from serving as a mean of classification, types also define behaviours of their instances. Consequently, types can define the following behaviours:
+
+__has [attribute type]__ - the ability to have an attribute of specified type attached to an instance
+
+__plays [role]__ - the ability to participate in relationships that allow for that role to be played
+
+__relates [role]__ (only relationships) - the ability of other instances to play this role in instances of this relationship type
+
+### Type Hierarchies
+Besides the modularity that the concept types provide, we are free to form subtype relations between concept types. For a given child concept type subtyping a parent concept type, the child concept type inherits the attributes owned and roles played by the parent type.
```

Comment:
Careful using _relations_ here, it means something different now, should be changed, I assumed this was talking only about subtyping relations